rules/windows/process_creation/win_susp_emotet_rudll32_execution.yml

```diff
@@ -18,7 +18,7 @@ detection:
             - '\rundll32.exe'
         CommandLine|endswith:
             - ',RunDLL'
-            - ',#1'
+            # - ',#1'  too generic - function load by ordinal is not Emotet specific
```

Comment:
I already tested on our environment, with EDR deploy on more than 2000 endpoints did not observe too much False Positive

Reply:
Hi, I am not concerned about false positives.
I meant that this is not specific to Emotet but many different threats. 

We already have a rule that covers DLL cals by ordinal. You could check if it matches what you're trying to cover. 
https://github.com/Neo23x0/sigma/blob/master/rules/windows/process_creation/win_susp_rundll32_by_ordinal.yml